modules/3d-tiles/src/lib/ion/ion.js

```diff
@@ -1,7 +1,6 @@
 // Minimal support to load tilsets from the Cesium ION services
 
-/* global fetch */
-import {_getErrorMessageFromResponse} from '@loaders.gl/core';
+import {fetchFile} from '@loaders.gl/core';
```

Comment:
Hmm... Isn't the main problem here that this module imports core, which modules generally are not supposed to do? I guess this PR relies on three-shaking out the non-fetchFile parts...

I suppose one way to decouple this function could take a custom `fetch` option and use `fetch` by default, and provide `fetchFile` in use for the better experience?

Reply:
As of now many modules still import various APIs from `core`, which is ok for now. My issue is that these imports must be limited and controlled, i.e. adding a new export, even experimental, to core must be very cautiously audited.